### DISCUSSION TITLE
Im slightly confused

### DISCUSSION THREAD

**Discussion by Orpheus013** [xCollateral/VulkanMod#236] [General]
So feather client comes with a bunch of built in mods right? Will this work with feather client if I remove sodium and other offending incompatibilities that I have added myself, not doing anything about the mods feather has built in?

**Comment by TheeStickmahn** [comment]
It's almost definitely not going to work.
VulkanMod fully changes the graphics API from OpenGL to Vulkan, making most mods that use OpenGL not work. Like. At all.
I think the cosmetics are using OpenGL.

**Comment by sqrvrt** [comment]
The basic answer is that if it's using custom ("sexy") GUI, it's not compatible.
